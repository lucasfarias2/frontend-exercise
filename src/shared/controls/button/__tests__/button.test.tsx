import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Button from '../button';

afterEach(cleanup);

describe('<Button />', () => {
  it('renders correctly', () => {
    const { container } = render(<Button type="submit" />);
    expect(container).toMatchSnapshot();
  });
});
