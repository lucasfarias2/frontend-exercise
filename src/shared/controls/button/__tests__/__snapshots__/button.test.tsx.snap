// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders correctly 1`] = `
<div>
  <button
    class="ui__button"
    type="submit"
  />
</div>
`;
